src/sentry/api/endpoints/data_export.py

```diff
@@ -32,9 +33,14 @@ def post(self, request, organization):
         if not features.has("organizations:data-export", organization):
             return Response(status=404)
 
+        limit = request.data.get("limit")
```

Comment:
`limit` can now be provided to specify the number of rows in the request (useful for API, the web client defaults to no limit)